server/sonar-search/src/main/java/org/sonar/search/EsSettings.java

```diff
@@ -156,10 +172,10 @@ private void configureCluster(ImmutableSettings.Builder builder) {
       }
     }
     builder.put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, replicationFactor);
-    builder.put("cluster.name", props.value(ProcessProperties.CLUSTER_NAME));
+    builder.put("cluster.name", getClusterName());
     builder.put("cluster.routing.allocation.awareness.attributes", "rack_id");
-    builder.put("node.rack_id", props.value(ProcessProperties.CLUSTER_NODE_NAME, "unknown"));
-    builder.put("node.name", props.value(ProcessProperties.CLUSTER_NODE_NAME));
+    builder.put("node.rack_id", getNodeName());
+    builder.put("node.name", getNodeName());
```

Comment:
you should use a variable to avoid risk of getting inconsistent values in name and rack_id (and the extra cost of call `props.value(...)`